unit/solvers/refinement/string_constraint_generator_valueof/calculate_max_string_length.cpp

```diff
@@ -114,26 +114,29 @@ SCENARIO("calculate_max_string_length",
     signedbv_typet(32),
     unsignedbv_typet(32),
     signedbv_typet(64),
-    unsignedbv_typet(64)};
+    unsignedbv_typet(64),
+  };
 
   for(const typet &type : int_types)
   {
-    WHEN(std::string("type = ")+type.pretty())
+    std::string type_str=type.pretty();
+    std::replace(type_str.begin(), type_str.end(), '\n', ' ');
+    WHEN("type = "+type_str)
```

Comment:
I don't think this is the best way to output extra info - Catch provides [logging macros](https://github.com/philsquared/Catch/blob/master/docs/logging.md) and there is support for using `INFO` on `irept`: (see `unit_tests.cpp`). (non-blocking)

Reply:
I think this is the WHEN for this test rather than just a logging message.